wordpress-mobile/WordPress-Android
# Stats Revamp: match data across platforms

This PR fixes data consistencies across Web, Android and iOS on Jetpack app

- Comparison between this week and last week excludes today's data
- Labels now read **7-days** instead of **Week**

Fixes #17731 

<img width=320 src="https://user-images.githubusercontent.com/990349/211978398-2b84d8e5-ca38-431e-abdc-64bf53b4282a.png" />

To test:

- Launch Jetpack app
- Navigate to Stats
- Verify the titles on the Views and Visitors card read as **Last 7-days** and **Previous 7-days**
- Verify Information below the graph uses the same like **7-days**
- Verify information on the Total Likes also reads like **7-days**
- Verify information on the Total Comments also reads **7-days** and not Week
- Also verify the Total count of Views, Visitors, Likes and Comments matches on the Web

>**Note**
> The Line graph on Views and Visitors card still shows data for today and matches Web
> There's a know issue around Unique Visitors, that's not related to this fix

## Regression Notes
1. Potential unintended areas of impact


2. What I did to test those areas of impact (or what existing automated tests I relied on)


3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:
